.travis.yml

```diff
@@ -48,11 +49,12 @@ aliases:
         - /c/Python36
         - /c/Python37
         - /c/Python38
+        - /c/Python39
         - /c/ProgramData/chocolatey/lib
         - /c/ProgramData/chocolatey/bin
         - /c/Users/travis/AppData/Local/pypoetry/Cache
     before_install:
-      - choco install python --version $JRNL_PYTHON_VERSION
+      - choco install python --version $JRNL_PYTHON_VERSION --pre
```

Comment:
Is `--pre` still useful if 3.9 isn't being added?

Reply:
done